src/Eshopworld.WorkerProcess/EShopworld.WorkerProcess.csproj

```diff
@@ -10,25 +10,29 @@
 
     <Platforms>x64</Platforms>
     <Platform>x64</Platform>
-    <Version>1.0.2</Version>
+    <Version>1.0.3</Version>
```

Comment:
I'm just thinking since you're changing the target framework, should the version be actually changed at least to 1.1.0?